### PR TITLE
Bugfix/982 Slow docker build for ARM

### PR DESCRIPTION
**Pull request by danigian**
# PR for issue #982

### CODE REVIEW

File: LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm32v7 [hunk 1]
```diff
@@ -1,32 +1,35 @@
...
 COPY --from=build /basicstation/deps/lgw/platform-rpi/reset_lgw.sh .
-COPY --from=build2 /basicstation/build-rpi-std/bin/station /bin/station.spispeed2
+COPY --from=build /basicstation/built/* .
 COPY LoRaEngine/modules/LoRaBasicsStationModule/station.conf .
-COPY LoRaEngine/modules/LoRaBasicsStationModule/start_basestation_arm.sh .
+COPY LoRaEngine/modules/LoRaBasicsStationModule/start_basestation.sh .
 COPY LICENSE .
-RUN chmod +x ./start_basestation_arm.sh
-ENTRYPOINT ["./start_basestation_arm.sh"]
+RUN chmod +x ./start_basestation.sh
```
Comment (Mandur) [inline LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm32v7:34]: Nit, you could rename those basicsstation

Comment (Mandur) [reply LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm32v7:34]: Not flagging other occurence

File: LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm32v7 [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
+# clean before making spi speed 2 version
 RUN make platform=rpi variant=std clean
 # make spi speed 2 version
 RUN sed -i "s|8000000|2000000|g" /basicstation/deps/lgw/platform-rpi/libloragw/src/loragw_spi.native.c
 RUN make platform=rpi variant=std
+RUN cp ./build-rpi-std/bin/station ./built/station.spispeed2
 
-FROM base
-WORKDIR /bin
+FROM arm32v7/debian:buster
```
Comment (Mandur) [inline LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm32v7:27]: I really like the idea of having a single dockerfile. However to retain compatibility with the iot edge toolkit i would suggest to edit the module.json to have the both architecture point to that file.

**Review by Mandur — APPROVED** [review]

File: LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm32v7 (hunk 2 above)
Comment (danigian) [reply LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm32v7:27]: As discussed, I will resolve, as the dockerfiles are anyways two 👍

**Pull request merged**